### ISSUE TITLE
Please support pipenv

### ISSUE THREAD

**Issue by gsemet** [pghilardi/atom-python-test#17]
Please support pipenv to start unit test without having to source virtualenv prior to launch atom.
thx

**Comment by jgirardet** [comment]
With latest version of pipenv you can lauch everything outside virtualenv 
juste do 
```
pipenv run atom .
```
it launches atom in virtualenv opening (with the  dot)  project folder
then  atom pytest works out of the box

**Comment by gsemet** [comment]
This is basically a command to launch atom from a virtualenv. My atom is started from desktop and carry several projects along side, and they don't share the same virtualenv. If we can find a way to describe per project how to source the virtualenv (just the `pipenv run pytest ...`) i would be happy. For the moment, it is too restricted, I cannot put `pipenv run` anywhere

**Comment by pghilardi** [comment]
@Stibbons,

I am the maintainer of the atom-python-virtualenv plug-in and I think that maybe it will suit to your use case. This plug-in allows changing virtualenv dynamically (it support virtualenvs from pipenv), without having to open atom with the env activated. Could you give a try? https://atom.io/packages/atom-python-virtualenv

I am planning a way to reload the virtualenv automatically based in the current project opened, but for now, it is not supported.

**Comment by gsemet** [comment]
Look promising. Do you support `pew` (virtualenv are created int `~/.local/virtualenvs/`, it is what pipenv uses for virtualenv storage

**Comment by pghilardi** [comment]
Yes. You can customize the default location of your envs home folder or even add additional envs paths.

**Comment by kasanitej** [comment]
@pghilardi Can you please do a video on how to use it.
